### ISSUE TITLE
scapholunate interval widening

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10716]
**Preferred term label:**
scapholunate interval widening

**Synonyms**

**Definition (free text, please give PubMed ID)**
Widening of the space between the scaphoid and lunate on x-ray; scapholunate dissociation; Terry Thomas sign

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Scapholunate advanced collapse (SLAC)
Scapholunate (SL) ligament injury
Perilunate dislocation (PLD)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001198
- label: Scapholunate interval widening
- parents: [Abnormal carpal morphology (HP:0001191)](https://hpo.jax.org/app/browse/term/HP:0001191)
- pmid: PMID:608295
- def: Widening of the space between the scaphoid and lunate on x-ray; scapholunate dissociation.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10716
- synonyms: Terry-Thomas sign(NONE)
